data/sailfish/hammerhead/resolutions.ini

```diff
@@ -0,0 +1,188 @@
+[image-high-16:9]
+capture = 1920x1080
+viewfinder=1280x720
+preview=960x720
+fps=15
+night = -1
+device=0
+mode=image
+
+[image-medium-16:9]
+capture = 1280x720
+viewfinder=1280x720
+preview=960x720
+fps=15
+night = -1
+device=0
+mode=image
+
+[image-high-4:3]
+capture = 3264x2448
+viewfinder=1280x960
+preview=960x720
+fps=15
+night = -1
+device=0
+mode=image
+
+[image-high1-4:3]
+capture = 2048x1536
+viewfinder=1280x960
+preview=960x720
+fps=15
+night = -1
+device=0
+mode=image
+
+[image-low-4:3]
+capture = 1024x768
+viewfinder=1024x768
+preview=960x720
+fps=15
+night = -1
+device=0
+mode=image
+
+[image-low-3:2]
```

Comment:
then kill this too? https://github.com/foolab/cameraplus/pull/40/files#diff-d2474976d06fa3f1aa7919745cb89347R181